ch.elexis.base.ch.arzttarife/src/ch/elexis/data/TarmedOptifier.java

```diff
@@ -209,7 +209,9 @@ public void clearContext(){
 			if (v.isInstance(code)) {
 				if (!tc.requiresSide()) {
 					newVerrechnet = v;
-					newVerrechnet.setZahl(newVerrechnet.getZahl() + 1);
+					if (!(",00.2530,00.2570,00.2550,00.2590,04.0620,04.1930,06.0430,06.0440,06.0730,06.0740,07.0300,24.0250,24.3250,28.0020,"
+							.contains("," + code.getCode())))
+						newVerrechnet.setZahl(newVerrechnet.getZahl() + 1);
```

Comment:
Wie wäre folgende Lösung
Set<String> codes = new HashSet<>(Arrays.asList("00.2530", etc));
und !codes.contains(code.getCode())
siehe https://stackoverflow.com/questions/2041778/how-to-initialize-hashset-values-by-construction